llvm/test/CodeGen/X86/apx/kmov-kk.ll

```diff
@@ -0,0 +1,14 @@
+; RUN: llc < %s -mtriple=x86_64-unknown -mattr=+avx512f,+egpr -show-mc-encoding | FileCheck --check-prefix=EGPR %s
+
+define <16 x i32> @kmovkk(ptr %base, <16 x i32> %ind, i16 %mask) {
+; EGPR: kmovq   %k1, %k2                        # EVEX TO VEX Compression encoding: [0xc4,0xe1,0xf8,0x90,0xd1]
```

Comment:
Is it possible to add a test to cover the memory fold case?

Reply:
It's not straightforward to me.  And I think the memory fold is already tested by ./llvm/test/TableGen/x86-fold-tables.inc